extensions/wikia/DesignSystem/scripts/DesignSystemGlobalFooter.js

```diff
@@ -0,0 +1,12 @@
+require(['jquery'],
+	function ($) {
+		if ($.cookie('useskin') === 'oasis')  {
+			$('.mobile-site-link').click(mobileSiteLinkClicked);
+			$('.mobile-site-link').css("display", "inline-block");
```

Comment:
We need to set it dynamically as we only show it when the cookie is set. That set, we can prepare a class in .scss, e.g. "mobile-site-link--active" that will set display: inline-block and we assign this class here, e.g.
```
.addClass ('mobile-site-link--active');
```

This way we keep all the presentation (styling) layer in the SCSS file :)